examples/lighting-app/nrfconnect/main/AppTask.cpp

```diff
@@ -426,13 +389,25 @@ void AppTask::StartBLEAdvertisementHandler(AppEvent * aEvent)
     }
 }
 
-#ifdef CONFIG_CHIP_NFC_COMMISSIONING
 void AppTask::ChipEventHandler(const ChipDeviceEvent * event, intptr_t /* arg */)
```

Comment:
This handler is added only if NFC is enabled, right? Probably you need to remove that condition.